src/Button/Button.features.stories.tsx

```diff
@@ -87,3 +87,28 @@ export const Small = () => <Button size="small">Default</Button>
 export const Medium = () => <Button size="medium">Default</Button>
 
 export const Large = () => <Button size="large">Default</Button>
+
+export const Loading = () => <Button loading>Default</Button>
+
+export const LoadingTrigger = () => {
+  const [isLoading, setIsLoading] = useState(false)
+
+  const handleClick = () => {
+    setIsLoading(true)
+    setTimeout(() => {
+      setIsLoading(false)
+    }, 3000)
+  }
+
+  return (
+    <Button loading={isLoading} onClick={handleClick} leadingIcon={DownloadIcon}>
+      Export
+    </Button>
+  )
+}
+
+export const LoadingCustomMessage = () => (
+  <Button loading loadingMessage="This is a custom loading message">
```

Comment:
Is it an sr-only message?

Reply:
Hmm good question. I think it is a good idea to indicate it in the name because I searched the code to see where we display the message and when we add this to the prop data it might not be obvious that it is sr only.

`hiddenLoadingMessage ` sounds like an option. `visuallyHiddenLoadingMessage` is too long. Is `sr-only` wording out-dated? 🤔 Could it be `srOnlyLoadingMessage`?

Reply:
Hi! 

The prop should ideally hint at what purpose it serves, but I think it doesn't have to describe implementation details. 

If `loadingMessage` feels misguiding as the message is not shown in the button, how do you feel about something along the lines of `loadingAnnouncement`? It signals a live region for screen readers without getting into what it doesn't do (visuallyHidden / srOnly)

Reply:
Love that @siddharthkp! Thank you 😄

Reply:
Love it too 😍